error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/JUnitSingleArgumentsTest.java

```diff
@@ -0,0 +1,30 @@
+package tech.picnic.errorprone.bugpatterns;
+
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.jupiter.api.Test;
+
+final class JUnitSingleArgumentsTest {
+  @Test
+  void identification() {
+    CompilationTestHelper.newInstance(JUnitSingleArguments.class, getClass())
+        .addSourceLines(
```

Comment:
I don't get what's not formatting-compliant here 🤔

Reply:
It was subtle; ran `./apply-error-prone-suggestions.sh` and pushed :)

Reply:
Ah thanks, didn't know that existed 😄 Also the command line error just gives 'did you mean' with the first few lines, which is confusing 🤪

Reply:
Yeah, that output is... "not great". Haven't really found a way yet to make that play nicer with Error Prone.